parastell/invessel_build.py

```diff
@@ -266,6 +316,34 @@ def angles_to_xyz(self, toroidal_angle, poloidal_angles, s, scale):
 
         return np.array(coords) * scale
 
+    def calculate_tangents(self, toroidal_angle, poloidal_angles, s, scale):
+        """Compute the tangents of a set of points, defined by a set of
+        poloidal angles, at a given toroidal angle. Takes s as a argument for
+        compatibility, but does nothing with it.
+
+        Arguments:
+            toroidal_angles (float): toroidal angle at which to evaluate
+                tangents [rad].
+            poloidal_angles (iterable of float): poloidal angles at which to
+                evaluate tangents [rad].
+            s (float): not used.
+            scale (float): a scaling factor between input and output data.
+
+        Returns:
+            (Nx3 numpy.array): array of poloidal tangents at each angle pair
+                specified.
+        """
+        eps = 1e-1
```

Comment:
This seems large - have you experimented with values between this and 1e-4?

Reply:
In my testing, 1e-1 gave the best results. Smaller values would result in asymmetry about the midplane for poloidal profiles that should be symmetric (i.e., beginning, midpoint, and end of period)